SOURCEBUILD.md

```diff
@@ -74,9 +80,50 @@ conda env update --name cugraph_dev --file conda/environments/cugraph_dev_cuda10
 conda activate cugraph_dev
 ```
 
-3) Build and install `libcugraph`. CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX`.
 
-  This project uses cmake for building the C/C++ library. To configure cmake, run:
+### Build and Install Using the `build.sh` Script
+Using the `build.sh` script make compiling and installig cuGraph a breeze.  To build and install, simply do:
+
+```bash
+$ cd $CUGRAPH_HOME
+$ ./build.sh clean
+$ ./build.sh libcugraph
+$ ./build.sh cugraph
+```
+
+There are several other options available on the build script for advanced users.
+`build.sh` options:
+```bash
+build.sh [<target> ...] [<flag> ...]
+   clean            - remove all existing build artifacts and configuration (start over)
+   libcugraph       - build the cugraph C++ code
+   cugraph          - build the cugraph Python package
+
+ and <flag> is:
+   -v               - verbose build mode
+   -g               - build for debug
+   -n               - no install step
+   --show_depr_warn - show cmake deprecation warnings
+   -h               - print this text
+
+examples:
+$ ./build.sh clean                        # remove prior build artifacts (start over)
+$ ./build.sh libcugraph -v                # compile and install libcugraph with verbose output
+$ ./build.sh libcugraph -g                # compile and install libcugraph for debug
+$ ./build.sh libcugraph -n                # compile libcugraph but do not install
+
+# make parallelism options can also be defined: Example build jobs to 4 (make -j4)
+$ PARALLEL_LEVEL=4 ./build.sh libcugraph
+
+Note that the libraries will be installed to the location set in `$PREFIX` if set (i.e. `export PREFIX=/install/path`), otherwise to `$CONDA_PREFIX`.
+```
+
+
+## Building each section independently
+#### Build and Install the C/CUDA `libcugraph` Library
```

Comment:
```suggestion
#### Build and Install the C++/CUDA `libcugraph` Library
```
Need to search for C/CUDA and replace C with C++

Reply:
done